[Bug] Adding ElytraFly to Muodule Helper list crashes upon entering server

### Describe the bug

Using the HUD editor to add Elytra Fly to the Module Helper (the thing with the on/off text, idk what its called) causes the game to crash. Attempting to use the HUD Editor or join a world instantly crashes the game. 

The error is as follows: 

(Joining world)
```
[13:34:03] [Render thread/FATAL]: Unreported exception thrown!
java.lang.NullPointerException: Cannot invoke "meteordevelopment.meteorclient.systems.modules.movement.elytrafly.ElytraFlightMode.getHudString()" because "this.currentMode" is null
	at meteordevelopment.meteorclient.systems.modules.movement.elytrafly.ElytraFly.getInfoString(ElytraFly.java:365) ~[meteor-client-0.4.7-1357.jar:?]
	at meteordevelopment.meteorclient.systems.modules.render.hud.modules.ModuleInfoHud.getModuleInfo(ModuleInfoHud.java:106) ~[meteor-client-0.4.7-1357.jar:?]
	at meteordevelopment.meteorclient.systems.modules.render.hud.modules.ModuleInfoHud.getModuleWidth(ModuleInfoHud.java:101) ~[meteor-client-0.4.7-1357.jar:?]
	at meteordevelopment.meteorclient.systems.modules.render.hud.modules.ModuleInfoHud.update(ModuleInfoHud.java:65) ~[meteor-client-0.4.7-1357.jar:?]
	at meteordevelopment.meteorclient.systems.modules.render.hud.HUD.onRender(HUD.java:167) ~[meteor-client-0.4.7-1357.jar:?]
	at meteordevelopment.orbit.listeners.LambdaListener.call(LambdaListener.java:83) ~[meteor-client-0.4.7-1357.jar:?]
	at meteordevelopment.orbit.EventBus.post(EventBus.java:47) ~[meteor-client-0.4.7-1357.jar:?]
	at net.minecraft.class_329.handler$bad000$onRender(class_329.java:2042) ~[intermediary-1.18.jar:?]
	at net.minecraft.class_329.method_1753(class_329.java:416) ~[intermediary-1.18.jar:?]
	at net.minecraft.class_757.method_3192(class_757.java:855) ~[intermediary-1.18.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1117) ~[intermediary-1.18.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:733) [intermediary-1.18.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:236) [intermediary-1.18.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:599) [fabric-loader-0.12.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.12.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.12.8.jar:?]
```

(Opening HUD Editor)
```
[13:37:21] [Render thread/FATAL]: Reported exception thrown!
net.minecraft.class_148: Rendering screen
	at net.minecraft.class_757.method_3192(class_757.java:883) ~[intermediary-1.18.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1117) ~[intermediary-1.18.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:733) [intermediary-1.18.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:236) [intermediary-1.18.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:599) [fabric-loader-0.12.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.12.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.12.8.jar:?]
Caused by: java.lang.NullPointerException: Cannot invoke "meteordevelopment.meteorclient.systems.modules.movement.elytrafly.ElytraFlightMode.getHudString()" because "this.currentMode" is null
	at meteordevelopment.meteorclient.systems.modules.movement.elytrafly.ElytraFly.getInfoString(ElytraFly.java:365) ~[meteor-client-0.4.7-1357.jar:?]
	at meteordevelopment.meteorclient.systems.modules.render.hud.modules.ModuleInfoHud.getModuleInfo(ModuleInfoHud.java:106) ~[meteor-client-0.4.7-1357.jar:?]
	at meteordevelopment.meteorclient.systems.modules.render.hud.modules.ModuleInfoHud.getModuleWidth(ModuleInfoHud.java:101) ~[meteor-client-0.4.7-1357.jar:?]
	at meteordevelopment.meteorclient.systems.modules.render.hud.modules.ModuleInfoHud.update(ModuleInfoHud.java:65) ~[meteor-client-0.4.7-1357.jar:?]
	at meteordevelopment.meteorclient.systems.modules.render.hud.HUD.onRender(HUD.java:167) ~[meteor-client-0.4.7-1357.jar:?]
	at meteordevelopment.meteorclient.gui.tabs.builtin.HudTab$HudScreen.method_25394(HudTab.java:307) ~[meteor-client-0.4.7-1357.jar:?]
	at net.minecraft.class_757.method_3192(class_757.java:874) ~[intermediary-1.18.jar:?]
	... 10 more
```

Full crash log: 
Joining sserver: https://www.toptal.com/developers/hastebin/zosefudola.sql
Opening HUD editor: https://www.toptal.com/developers/hastebin/wemecisewa.sql

### Steps to reproduce

```bash
- Go to HUD editor
- Add Elytra Fly module to the Module Helper list
```


### Meteor Version

Dev Build 1357 (with Meteor Rejects, Sodium, Lithium, SmoothBoot and Multiconnect)

### Minecraft Version

1.18

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

--- a/src/main/java/meteordevelopment/meteorclient/systems/modules/movement/elytrafly/ElytraFly.java
+++ b/src/main/java/meteordevelopment/meteorclient/systems/modules/movement/elytrafly/ElytraFly.java
@@ -208,7 +208,7 @@ public class ElytraFly extends Module {
         .build()
     );
 
-    private ElytraFlightMode currentMode = new Vanilla();;
+    private ElytraFlightMode currentMode = new Vanilla();
 
     public ElytraFly() {
         super(Categories.Movement, "elytra-fly", "Gives you more control over your elytra.");

--- a/src/main/java/meteordevelopment/meteorclient/systems/modules/movement/elytrafly/ElytraFly.java
+++ b/src/main/java/meteordevelopment/meteorclient/systems/modules/movement/elytrafly/ElytraFly.java
@@ -208,7 +208,7 @@ public class ElytraFly extends Module {
         .build()
     );
 
-    private ElytraFlightMode currentMode;
+    private ElytraFlightMode currentMode = new Vanilla();;
 
     public ElytraFly() {
         super(Categories.Movement, "elytra-fly", "Gives you more control over your elytra.");

